node/src/actors/chain_manager/handlers.rs

```diff
@@ -89,6 +91,11 @@ impl Handler<EpochNotification<EveryEpochPayload>> for ChainManager {
         let current_epoch = msg.checkpoint;
         self.current_epoch = Some(current_epoch);
 
+        // Set the right Rescue Committee according to the Epoch
+        self.chain_state
+            .superblock_state
+            .set_rescue_committe(get_rescue_committee(current_epoch, get_environment()));
```

Comment:
There may be a race condition here. If we receive some votes before setting the rescue committee they will be ignored.

Reply:
The consolidation of the superblock is in epochs finished in 0, and the first epoch to sign a superblockvote is in epochs finished in 1, so if the change of one committee to another is in an epoch finished in 0, it should not be a problem

Reply:
Maybe this could be fixed by setting the committee when the chain manager starts (just like it was being set before), and also setting it here.

Reply:
I was thinking about the case when the node starts and receives some votes before the first epoch notification.

Reply:
The problem here is that we obtain the epoch information in the EpochNotification handler